Readme.md

```diff
@@ -1,39 +1,66 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128533541/16.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T150957)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView- How to conditionally enable and disable the batch editing on the client side
+# Grid View for ASP.NET Web Forms - How to control data editing on the client side in batch edit mode
```

Comment:
I'd like to question your choice of words here.

> How **to control** data editing

The word "to control" implies much, it can be actually anything. You can modify user input on-the-fly, you can prevent users from editing something, etc. However, this topic covers a very specific scenario, namely when you forbid users from editing a cell based on the current check box state. This topic does not explain how to assume the general control over user input.

The original topic name was good and bad at the same time. We could enhance it instead:

```suggestion
# Grid View for ASP.NET Web Forms - How to enable or disable the cell edit functionality in batch mode based on a condition
```

This is just a suggestion. You can come up with something better instead.

Reply:
I like your variant and absolutely agree with your comments. Since this scenario is a specific one, we should tell a user about it. Thank you!

Reply:
You are always welcome. Every time you use a word, you should ask yourself -- is this word clear and unambiguous? Does it serve its purpose well in the current context? If you can answer "yes" to the both questions -- it means you've chosen the right word for the task. If not -- it's probably worth taking another look.